orchestrator/careplanservice/service.go

```diff
@@ -89,27 +90,24 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 	mux.HandleFunc("POST /cps/Task", auth.Middleware(authConfig, func(writer http.ResponseWriter, request *http.Request) {
 		err := s.handleCreateTask(writer, request)
 		if err != nil {
-			// TODO: proper OperationOutcome
 			log.Info().Msgf("CarePlanService/CreateTask failed: %v", err)
-			http.Error(writer, "Create Task at CarePlanService failed: "+err.Error(), http.StatusBadRequest)
+			s.writeOperationOutcomeFromError(err, "CarePlanService/CreateTask", writer)
 			return
 		}
 	}))
 	mux.HandleFunc("PUT /cps/Task/{id}", auth.Middleware(authConfig, func(writer http.ResponseWriter, request *http.Request) {
 		err := s.handleUpdateTask(writer, request)
 		if err != nil {
-			// TODO: proper OperationOutcome
 			log.Info().Msgf("CarePlanService/UpdateTask failed: %v", err)
-			http.Error(writer, "Update Task at CarePlanService failed: "+err.Error(), http.StatusBadRequest)
+			s.writeOperationOutcomeFromError(err, "CarePlanService/UpdateTask", writer)
 			return
 		}
 	}))
 	mux.HandleFunc("POST /cps/CarePlan", auth.Middleware(authConfig, func(writer http.ResponseWriter, request *http.Request) {
 		err := s.handleCreateCarePlan(writer, request)
 		if err != nil {
-			// TODO: proper OperationOutcome
-			log.Info().Msgf("CarePlanService/CarePlan failed: %v", err)
-			http.Error(writer, "Create CarePlan at CarePlanService failed: "+err.Error(), http.StatusBadRequest)
+			log.Info().Msgf("CarePlanService/CreateCarePlan failed: %v", err)
```

Comment:
Can you move these info log messages to the newly introduced function? Since it's such a repeating pattern.